.github/workflows/node-pr-jobs-secure.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright Strimzi authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+name: Node
+
+# Running target makes it secure as this is the only way to get hold of github secrets
+on: [pull_request_target]
+
+jobs:
+  build-current:
+    runs-on: ubuntu-latest
+    name: Build and Test
+    steps:
+      - name: Get PR head
+        id: pr_head
+        run: |
+          OWNER=$(jq -r '.pull_request.base.repo.owner.login' "$GITHUB_EVENT_PATH")
```

Comment:
Is it a guarantee we will always have jq from `ubuntu-latest`?

Reply:
Good point, might be worth changing these to an LTS release